README.md

```diff
@@ -1,20 +1,20 @@
-# Project Name
+# Odin-youtube-clone
 
-> One paragraph statement about the project.
+> This project is about embedding videos,images and using flexbox properties for positioning and also media query for responsiveness.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./odin-yt-clone.png)
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- Hyper-text mark-up language
```

Comment:
Use HTML and CSS instead, also you can then highlight what part of HTML used. e.g.

```markdown
- HTML
   - Semantic HTML
   - HTML forms
- CSS
   - CSS Flexbox for effective layout
   - CSS Media query for responsiveness
   - CSS Positioning
- [Prettier](https://prettier.io/) for code formatting
- [Styelint](https://stylelint.io/) for CSS linting
- [HTML Validator](https://validator.w3.org/) and [Webhint](https://webhint.io/)
- Github Actions for workflow
- Netlify for deployment 
```

No need for writing markdown, you only used markdown for the readme.

Reply:
Thanks for the idea, Appreciate it!